x-pack/plugins/maps/public/components/layer_panel/settings_panel/settings_panel.js

```diff
@@ -13,13 +13,17 @@ import {
   EuiPanel,
   EuiFormRow,
   EuiFieldText,
-  EuiRange,
   EuiSpacer,
-  EuiCallOut,
+  EuiCallOut
 } from '@elastic/eui';
+
 import { ValidatedRange } from '../../../shared/components/validated_range';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { ValidatedDualRange } from 'ui/validated_range';
```

Comment:
There is one error case that `ValidatedDualRange` does not cover that that is allowing both lower and upper values to be empty. You are going to need to update `ValidatedDualRange` with `allowEmptyRange` prop that will be true by default so it does not break the RangeControl. Then, here, you will set that to false.

Here is the error state that is created when both lower and upper values are set to empty string

<img width="500" alt="Screen Shot 2019-03-13 at 9 45 08 AM" src="https://user-images.githubusercontent.com/373691/54293050-de52c500-4574-11e9-8c21-57bd1a7915e4.png">